### PR TITLE
Fix rossystem parser: correct the order and parse "FromRosNode"

### LINKED ISSUE #36: Rossystem parser cannot parse system file with "FromRosNode"

### LINKED ISSUE #38: Rossystem parser should parse Rosparameters at the end

### PR DESCRIPTION
**Pull request by ipa-rwu**
closes #36 
closes #38

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -13,14 +13,16 @@ catkin_package()
 if (CATKIN_ENABLE_TESTING)
   find_package(roslaunch REQUIRED)
   roslaunch_add_file_check(tests/model_generators_roslaunch_test.xml)
+  find_package(rostest REQUIRED)
+  add_rostest(tests/model_generators_roslaunch_test.xml)
```
Comment (ipa-nhg) [inline CMakeLists.txt:17]: we plan to do this repo ROS-agnostic, actually I am using it as a python module installed with the "pip" command.
The reason is that it has to work for ROS1 and ROS2, then probably these lines will be removed soon.

**Review by ipa-nhg — APPROVED** [review]
Thanks a lot for this fix @ipa-rwu  👍🏽 

I tested the version locally and it works! could you please merge here also the update of the example https://github.com/ipa-rwu/ros_model_parser/pull/1 ? then we have everything on the same PR. Thanks!

File: CMakeLists.txt (hunk 1 above)
Comment (ipa-rwu) [reply CMakeLists.txt:17]: same for me. I also use it as a python module. As you can find in [gitlab]( https://gitlab.cc-asp.fraunhofer.de/ipa326/ambs_creator/-/blob/main/install.sh)

I updated the example

File: resources/robotino.rossystem [hunk 2]
```diff
@@ -1,38 +1,41 @@
-RosSystem { Name 'robotino' 
-    RosComponents ( 
-        ComponentInterface { name 'joy'
+RosSystem { Name 'robotino'
+    RosComponents (
+        ComponentInterface { name 'joy_node' FromRosNode 'joy'
```
Comment (ipa-nhg) [inline resources/robotino.rossystem:3]: This is not valid, the node ''joy' doesn't exists.

See please: https://github.com/ipa-rwu/ros_model_parser/pull/1/files

**Pull request closed without merging**